src/Approvals/SFA.DAS.Approvals/Application/Cohorts/Commands/CreateCohort/CreateCohortCommandHandler.cs

```diff
@@ -39,7 +39,6 @@ public async Task<CreateCohortResult> Handle(CreateCohortCommand request, Cancel
                     AccountId = request.AccountId,
                     AccountLegalEntityId = request.AccountLegalEntityId,
                     CourseCode = request.CourseCode,
-                    ProviderId = request.ProviderId,
```

Comment:
I can see ProviderId is optional on the Reservations side, but there is a uint cast on the following line which may cause this to fail or pass a zero through, not sure what would happen if it's zero

https://github.com/SkillsFundingAgency/das-apim-endpoints/blob/master/src/Approvals/SFA.DAS.Approvals/Services/AutoReservationsService.cs#L63